Makefile

```diff
@@ -1079,3 +1085,18 @@ show_release_notes:
 .PHONY: spellcheck
 spellcheck:
 	bin/spell_check_java.sh
+
+$(JARS_JACOCO_AGENT):
+	mkdir -p $(@D)
+	curl $(JARS_JACOCO_AGENT_URL) --output $@
+	unzip $(JARS_JACOCO_AGENT) jacocoagent.jar -d $(BUILD_DIR)/jars
+
+$(JARS_JACOCO_CLI):
+	mkdir -p $(@D)
+	curl $(JARS_JACOCO_CLI_URL) --output $@
+
+coverage: $(FZ_INT) $(FZ_C) $(MOD_TERMINAL) $(MOD_JAVA_BASE) $(BUILD_DIR)/tests $(JARS_JACOCO_AGENT) $(JARS_JACOCO_CLI)
```

Comment:
Please add a comment to explain what this does: running the tests both interpreted and with the C backend to collect coverage date using jacoco and then create html to show the results in a browser. 

In a later step, we might want to enable this via an option or env var given to `fz` such that we can easily collect coverage data.